### ISSUE TITLE
get Revl at one of the shows

### ISSUE THREAD

**Issue by markdav-is** [EugTech/king-pong-plan-2019#37]

**Comment by jessicabob** [comment]
@nohorse - this is happening in August, yes?

**Comment by markdav-is** [comment]
Yes!  That's a great talking point.  Revl will be there and we need to provide them two mic stands.  We also need to illuminate the controller a bit.  I have some budget LEDS that will do the trick.  I'll start that convo in the game-update slack channel.

**Comment by jessicabob** [comment]
that sounds cool - I think that if they will let us we should make a blurb and put it in the event marketing.

**Comment by markdav-is** [comment]
ordered some trypods, weights, and go-pro mounts for the Revl cameras.
